frontend/src/components/core/Block/Block.tsx

```diff
@@ -39,6 +40,7 @@ import {
 } from "./styled-components"
 
 const ExpandableLayoutBlock = withExpandable(LayoutBlock)
+const TabContainer = Tabs(LayoutBlock)
```

Comment:
Hm, I'm not sure which one would be preferable. I think I'm leaning toward wanting to stick with the `with*` pattern because it looks like it's a relatively common pattern for hooks that are higher order components, but I don't have a strong opinion on this

Reply:
I agree that `with` would be the appropriate prefix here, what I'm more concerned about is that the expander is grouped into the `hoc` folder. I refactored the Tabs component a bit so that this higher-order component pattern is not even required. I think that we might want to do the same with expander as well so that it is also consistent with 	`Form`.